common/Labs.MultiTarget.props

```
@@ -53,4 +53,14 @@
     <EmbeddedResource Remove="obj\bin\**" />
     <UpToDateCheckInput Remove="obj\bin\**" />
   </ItemGroup>
+
+  <!-- Workaround for Mac: https://developercommunity.visualstudio.com/t/XamarinMac-binaries-are-missing-in-173/10164443#T-N10164676 10/6/22 -->
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('Xamarin.Mac')) ">
```

See here:
https://github.com/CommunityToolkit/Labs-Windows/blob/ed6bd23523770f34ebf6b41304dd257374fe1f04/common/Labs.TargetFrameworks.All.props#L11

```suggestion
  <ItemGroup Condition=" $(TargetFramework.StartsWith('xamarinmac')) ">
```